creator-node/src/models/index.js

```diff
@@ -18,6 +20,23 @@ const sequelize = new Sequelize(config.get('dbUrl'), {
     min: 5,
     acquire: 60000,
     idle: 10000
+  },
+  dialectOptions: {
+    // number of milliseconds before a statement in query will time out, default is no timeout
+    statement_timeout: STATEMENT_TIMEOUT,
+
+    // number of milliseconds before a query call will timeout, default is no timeout
+    // query_timeout: QUERY_TIMEOUT,
+
+    // number of milliseconds to wait for connection, default is no timeout
+    // connectionTimeoutMillis: 1000
+
+    // number of milliseconds before terminating any session with an open idle transaction, default is no timeout
+    // idle_in_transaction_session_timeout: 1000
+    options: {
```

Comment:
what's this options property for?

Reply:
can't find it in docs, removed

Reply:
ah it exists in latest sequelize
https://github.com/sequelize/sequelize/blob/main/src/dialects/postgres/connection-manager.js